apps/site/assets/ts/components/Accordion.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from "react";
+
+// A simple accordion implementation, based on site_web/templates/partial/_accordion_ui
+interface Props {
+  id: string;
+  title: string;
+  children: ReactElement<HTMLElement>;
+}
+
+const Accordion = (props: Props): ReactElement<HTMLElement> => {
```

Comment:
Could you please add a quick test for this component, just rendering it to a snapshot?